src/morphocut/image.py

```diff
@@ -95,6 +95,15 @@ class FindRegions(Node):
 
     .. _skimage.measure.regionsprops: https://scikit-image.org/docs/dev/api/skimage.measure.html
 
+    Args:
+        image (np.ndarray or Variable[np.ndarray]): An image whose mask we have to find region with.
```

Comment:
Don't repeat the type in Variable[...]. It is obvious that the Variable has to have the same type as the raw value. Just write `np.ndarray or Variable`.

This comment applies to all docstrings.

Reply:
I don't see this resolved. This is what I want:

```
image (np.ndarray or Variable): ...
```